src/coord/src/catalog/builtin.rs

```diff
@@ -423,6 +423,24 @@ pub const TYPE_BPCHAR: BuiltinType = BuiltinType {
     pgtype: &postgres_types::Type::BPCHAR,
 };
 
+pub const TYPE_CHAR_ARRAY: BuiltinType = BuiltinType {
+    schema: PG_CATALOG_SCHEMA,
+    id: GlobalId::System(1043),
```

Comment:
Can you re-run `verify_function_sanity` to double check the new oids?

Reply:
Ah, I had done this already! I'll make sure to make more explicit mention in my following PRs.

Reply:
:+1: